src/app/betting/feature/accept-bets-page.component.ts

```diff
@@ -0,0 +1,176 @@
+import { PlayerService } from '../../shared/data-access/player.service';
+import { TableModule } from 'primeng/table';
+import { NgOptimizedImage } from '@angular/common';
+import { ButtonModule } from 'primeng/button';
+import { RouterLink } from '@angular/router';
+import {
+  BetStatus,
+  showMessageOnError,
+} from '../../shared/util/supabase-helpers';
+import { StronglyTypedTableRowDirective } from '../../shared/ui/strongly-typed-table-row.directive';
+import { BetService } from '../../shared/data-access/bet.service';
+import { MessageService } from 'primeng/api';
+
+@Component({
+  selector: 'joshies-place-bet-choose-player-page',
+  standalone: true,
+  imports: [
+    PageHeaderComponent,
+    HeaderLinkComponent,
+    TableModule,
+    NgOptimizedImage,
+    ButtonModule,
+    RouterLink,
+    StronglyTypedTableRowDirective,
+  ],
+  template: `
+    <joshies-page-header headerText="Accept Bets" alwaysSmall>
+      <joshies-header-link
+        text="GM Tools"
+        routerLink=".."
+        chevronDirection="left"
+      />
+    </joshies-page-header>
+
+    @if (displayBets(); as bets) {
+      <p-table
+        [value]="bets"
+        [defaultSortOrder]="-1"
+        sortField="score"
+        [sortOrder]="-1"
+        [scrollable]="true"
+      >
+        <ng-template pTemplate="header">
+          <tr>
+            <th style="width: 60%;"></th>
+            <th></th>
+          </tr>
+        </ng-template>
+        <ng-template
+          pTemplate="body"
+          [joshiesStronglyTypedTableRow]="bets"
+          let-bet
+        >
+          <tr>
+            <!-- Bet Terms -->
+            <td>
+              <div class="flex flex-column gap-2 -py-2">
+                <div>
+                  {{ bet.requesterName }} wagers:
+                  {{ bet.requesterWager }}
+                </div>
+                <div>
+                  {{ userNameAndScore() }} wagers:
+                  {{ bet.yourWager }}
+                </div>
+                <div>{{ bet.description }}</div>
+              </div>
+            </td>
+            <!-- Accept/Reject Buttons -->
+            <td>
+              <div
+                class="text-right flex gap-2 flex-column md:flex-row justify-content-end"
+              >
+                <p-button
+                  label="Accept Bet"
+                  icon="pi pi-check"
+                  styleClass="w-full"
+                  (onClick)="acceptBet(bet.id)"
+                  [disabled]="
+                    bet.yourWager > (userPlayer()?.score ?? 0) ||
+                    bet.requesterWager > (bet.requesterScore ?? 0)
+                  "
+                  [loading]="submitting()"
+                />
+                <p-button
+                  label="Reject Bet"
+                  icon="pi pi-times"
+                  styleClass="w-full"
+                  (onClick)="rejectBet(bet.id)"
+                  [loading]="submitting()"
+                />
```

Comment:
To add more contrast, we can add `severity` to the buttons to make them green or red:

```html
<p-button
  label="Accept"
  severity="success"
  icon="pi pi-check"
  styleClass="w-full"
  (onClick)="acceptBet(bet.id)"
  [disabled]="
    bet.yourWager > (userPlayer()?.score ?? 0) ||
    bet.requesterWager > (bet.requesterScore ?? 0)
  "
  [loading]="submitting()"
/>
<p-button
  label="Reject"
  severity="danger"
  icon="pi pi-times"
  styleClass="w-full"
  (onClick)="rejectBet(bet.id)"
  [loading]="submitting()"
/>
```